htlcswitch/link.go

```diff
@@ -1853,11 +1814,14 @@ func (l *channelLink) handleUpstreamMsg(msg lnwire.Message) {
 			return
 		}
 
-		// If there are pending local updates, try to update the commit
-		// tx. Pending updates could already have been present because
-		// of a previously failed update to the commit tx or freshly
-		// added by processRemoteAdds.
-		if l.channel.PendingLocalUpdateCount() > 0 {
+		// The revocation window opened up. If there are pending local
+		// updates, try to update the commit tx. Pending updates could
+		// already have been present because of a previously failed
+		// update to the commit tx or freshly added in by
+		// processRemoteAdds. Also in case there are no local updates,
+		// but there are still remote updates that are not in the remote
+		// commit tx yet, send out an update.
+		if l.channel.OweCommitment() {
```

Comment:
nit: should this change really be in a separate commit? Seems orthogonal to removing the ticker.

Reply:
It is in the same commit because the log commit ticker was 'fixing' the problem caused by just checking `l.channel.PendingLocalUpdateCount() > 0`.

Reply:
Definitely not a blocker, was just curious.